src/MoBi.Assets/MoBi.Assets.csproj

```diff
@@ -26,9 +26,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="11.1.62" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="11.1.62" />
-    <PackageReference Include="OSPSuite.Core" Version="11.1.62" />
+    <PackageReference Include="OSPSuite.Assets" Version="11.1.63-pxkrujcl" />
+    <PackageReference Include="OSPSuite.Assets.Images" Version="11.1.63-pxkrujcl" />
+    <PackageReference Include="OSPSuite.Core" Version="11.1.63-pxkrujcl" />
```

Comment:
why referencing this Core version from the **PR**?
11.1.64 is available already. 
And actually we could merge https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1803 first and then update directly to 11.1.65

Reply:
done